package.json

```diff
@@ -26,36 +26,31 @@
     "@actions/github": "^5.0.3",
     "@babel/helper-plugin-utils": "^7.18.6",
     "@babel/plugin-syntax-function-bind": "^7.18.6",
-    "@babel/register": "^7.18.6",
-    "@babel/runtime": "^7.18.6",
-    "@gera2ld/locky": "^0.1.1",
-    "@gera2ld/plaid": "~1.5.6",
-    "@gera2ld/plaid-vue": "~1.6.5",
-    "@gera2ld/plaid-webpack": "~1.5.5",
+    "@gera2ld/plaid": "~2.5.6",
+    "@gera2ld/plaid-common-ts": "~2.5.1",
+    "@gera2ld/plaid-test": "^2.5.0",
+    "@gera2ld/plaid-webpack": "~2.5.3",
+    "@gera2ld/plaid-webpack-vue3": "~2.5.5",
     "@types/chrome": "^0.0.191",
     "@types/firefox-webext-browser": "94.0.1",
     "amo-upload": "^0.2.0",
     "cross-env": "^7.0.3",
     "cross-spawn": "^7.0.3",
     "del": "^6.1.1",
-    "eslint-plugin-vue": "^9.3.0",
+    "eslint-plugin-jest": "^26.7.0",
     "fancy-log": "^2.0.0",
     "gulp": "^4.0.2",
     "gulp-plumber": "^1.1.0",
-    "husky": "^4.2.3",
+    "husky": "^8.0.1",
     "js-yaml": "^4.1.0",
     "jsdom": "^20.0.0",
-    "npm-run-all": "^4.1.5",
     "plugin-error": "^2.0.0",
+    "postcss-scss": "^4.0.4",
+    "postcss-simple-vars": "^6.0.3",
     "sharp": "0.30.1",
```

Comment:
Just curious, why is the version of `sharp` locked?

Reply:
Because transparency+resize is bugged in all subsequent versions. I've reported the bug upstream.